ui/brave_ads/message_popup_view.cc

```diff
@@ -89,18 +83,32 @@ MessagePopupView::MessagePopupView(const Notification& notification)
   views::Widget::InitParams params(views::Widget::InitParams::TYPE_POPUP);
   params.type = views::Widget::InitParams::TYPE_WINDOW_FRAMELESS;
   params.z_order = ui::ZOrderLevel::kFloatingWindow;
-  const gfx::Size screen_size =
-      display::Screen::GetScreen()->GetPrimaryDisplay().size();
+
+  const gfx::Rect visible_frame = GetVisibleFrameForPrimaryDisplay();
+
+  const int x = visible_frame.x() + visible_frame.width()
```

Comment:
there is `Rect::right()` - it's equal to `x() + width()`

Reply:
Fixed